README.md

```diff
@@ -37,12 +39,47 @@ Task items and progress bars correspond with your chosen dashboard colors and po
     - To change your dashboard courses, go to **Courses** in the left sidebar, go to **All Courses** and star the classes that you want on your dashboard.
 - The **Unfinished** assignments list will show all assignments from the dashboard courses that are both unsubmitted and ungraded or have a grade of 0.
 
+## Frequently Asked Questions (FAQ)
+
+### How does this extension sync with the Canvas account?
+
+The extension utilizes Canvas API to fetch assignment data directly from your Canvas account. It securely accesses your course information and assignment details to provide you with up-to-date task lists and progress tracking.
+
+### How can I customize the appearance of the extension?
+
+Yes, you can customize the appearance of the extension to match your preferences. The task items and progress bars are designed to correspond with your chosen dashboard colors and positions in Canvas.
+
+### What platforms are supported for this extension?
+
+Currently, the extension is available for Chrome and Firefox browsers. You can download it from the respective web stores or repositories.
+
+### Why don't I see all of my courses on the dashboard?
+
+This extension only displays courses that have assingments. If a course has no assignments, it will not show on the extension.
+
+### How accurate is the progress tracking?
+
+The progress bars in the extension provide an indication of your completion status for assignments within each course. However, please note that any changes made within Canvas (such as new assignments or updates) may take some time to reflect accurately in the extension.
```

Comment:
note that 0-point assignments don't count as progress, new assignments will only show after reloading